pkg/cluster/internal/create/actions/createworker/gcp.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package createworker
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"net/url"
+)
+
+type GCPBuilder struct {
+	capxProvider string
+	capxName     string
+	capxTemplate string
+	capxEnvVars  []string
+}
+
+func newGCPBuilder() *GCPBuilder {
+	return &GCPBuilder{}
+}
+
+func (b *GCPBuilder) setCapxProvider() {
+	b.capxProvider = "gcp:v1.2.1"
```

Comment:
Please add this version to the DEPENDENCIES file (in core.gcp.cluster-api-gcp + core.gcp.cluster-api-gcp-templates)

Reply:
Done